nm/src/nm.c

```diff
@@ -494,8 +494,10 @@ _j4status_nm_device_property_changed(NMDevice *device, GParamSpec *pspec, gpoint
             g_object_unref(section->ap);
         }
         section->ap = nm_device_wifi_get_active_access_point(NM_DEVICE_WIFI(device));
-        if ( section->ap != NULL )
+        if ( section->ap != NULL ) {
+            g_object_ref(section->ap);
             section->strength_handler = g_signal_connect(g_object_ref(section->ap), "notify::strength", G_CALLBACK(_j4status_nm_access_point_property_changed), section);
```

Comment:
Here the `g_object_ref()` is done already. You need to stick to one, don’t mind either, though I used inline one to avoid bracket back then.